stacks/terraform/vpc/sharing.tf

```diff
@@ -0,0 +1,3 @@
+output "vpc_id" {
+  value = module.vpc.vpc_id
+}
```

Comment:
Since I am not too familiar with Terramate yet, I am just wondering. Which one https://github.com/terramate-io/terramate-quickstart-aws/pull/203/files#diff-dc0f18a94f4273afd42721dc7a41896b318984b7751967b6cd4d579836adbce9R3 is referring to. Is it stacks/terraform/vpc/outputs.tm or this output here?

Reply:
Hi @poberherr, 

This is a WIP feature, probably landing on the upcoming v0.10.1 release. It is still undocumented and some details can change before the release.

But about your question, yes, this file is generated from `output` and `input` blocks defined in a `.tm` file, in this case it is `stacks/terraform/vpc/outputs.tm`. The `sharing_backend.filename` attribute configures the generated file name.